draft-marchan-kdl2.md

```diff
@@ -397,18 +437,29 @@ interpreted as described in the following table:
 | Unicode Escape                | `\u{(1-6 hex chars)}` | Code point described by hex characters, as long as it represents a [Unicode Scalar Value](https://unicode.org/glossary/#unicode_scalar_value) |
```

Comment:
Is there anything special RFCs typically expect about external links like these?

Reply:
Most of these should eventually be [references](https://github.com/cabo/kramdown-rfc/wiki/Syntax2#external-references). It takes a bit of research to write each reference, so I propose to do that as a separate PR.